streamly-coreutils.cabal

```diff
@@ -0,0 +1,71 @@
+cabal-version:       2.4
+-- Initial package description 'streamly-coreutils.cabal' generated by
+-- 'cabal init'.  For further documentation, see
+-- http://haskell.org/cabal/users-guide/
+
+name:                streamly-coreutils
+version:             0.1.0.0
+-- synopsis:
+description:
+   Port useful commands from the GNU `coreutils` to Haskell functions using
+   streamly.
+
+   GNU `coreutils` are commonly used, well known interfaces to perform common
+   tasks on POSIX systems. It will be useful to have Haskell implementations of
+   these utilities in general and also to utilize the knowledge of these utilities
+   gained by programmers, admins, script writers for writing Haskell scripts.
+
+   Some of these, but not all, are available in some existing Haskell libraries in
+   some form or other. However, there is no comprehensive collection available,
+   performance may not be at par and implementations may not be streaming. The
+   goal of this project is to have a comprehensive streaming collection using
+   Streamly with performance competitive with that of the C implementations
+   where it matters.
+
+   One may ask why not just use fork the installed binaries and use those. In many
+   cases it may make sense to fork the installed  utilities to achieve the task.
+   However, in many other cases it may be better to use a streaming function
+   instead of forking a process and parsing the output.
+
+homepage:            https://github.com/composewell/streamly-coreutils
+bug-reports:         https://github.com/composewell/streamly-coreutils/issues
+license:             BSD-3-Clause
+
+license:             BSD-3-Clause
+license-file:        LICENSE
+author:              Shruti Umat
+maintainer:          streamly@composewell.com
+copyright:           2020 Composewell Technologies
+-- category:
+extra-source-files:  CHANGELOG.md, README.md
+
+library
+  build-depends:
```

Comment:
Use `ghc-options: -Wall`

Reply:
It's present in `common compile-options` (Line 70)

Reply:
You need to import that here to make it effective. Also pull those definitions before the library stanza so that we follow define before use convention.